### ISSUE TITLE
Improvement on Beta download page 

### ISSUE THREAD

**Issue by Emily-Jiang** [OpenLiberty/openliberty.io#2579]
On the beta download page, there is not much information on the beta. I think it is a good idea if we can add a link to point to the beta blog post so that people to browse the blog to find out the beta content.

**Comment by Emily-Jiang** [comment]
I thought this further, we can do the same thing for GA release as well. Thoughts?

**Comment by Sreejith-Websphere** [comment]
@Emily-Jiang @kinueng @aknguyen7 can we have version eg: 22.0.0.5-beta act as a link to beta blog post https://openliberty.io/blog/2022/04/13/22005-beta.html, so each beta version has a link to their respective blog post, Need input
<img width="1792" alt="Screenshot 2022-05-09 at 2 53 54 PM" src="https://user-images.githubusercontent.com/87368885/167383144-ecc48c48-f218-4621-9458-eb18d1e9c1f2.png">

**Comment by Emily-Jiang** [comment]
> @Emily-Jiang @kinueng @aknguyen7 can we have version eg: 22.0.0.5-beta act as a link to beta blog post https://openliberty.io/blog/2022/04/13/22005-beta.html, so each beta version has a link to their respective blog post, Need input <img alt="Screenshot 2022-05-09 at 2 53 54 PM" width="1792" src="https://user-images.githubusercontent.com/87368885/167383144-ecc48c48-f218-4621-9458-eb18d1e9c1f2.png">

I think it is a good idea, maybe just add one more column.

**Comment by kinueng** [comment]
Hi @Emily-Jiang,

The Beta downloads page has a link to all the blogs related to the beta builds. The sentence `Beta content may change from release to release` is a link to the blog posts with the `beta` tag. This is the link https://openliberty.io/blog/?search=beta&key=tag.

The top blog post filtered by beta tag is for `22.0.0.5-beta`. 

Would it help if we rework the `Beta content may change from release to release` to be more obvious the link takes you to beta blog posts?

Hi @Sreejith-Websphere,

The other tables do not have the UX where the version is a link. I think it will be out of place to only have the `22.0.0.5-beta` be a link without any description on why it is a link (e.g. "This is a link to blog posts"). I think promoting the existing link to filter blog posts by `beta` tag is effective and easiest to maintain from a coding perspective.

**Comment by kinueng** [comment]
For the short term improvement, we can rework the existing wording on the beta view. Long term, we will use #2660.

Hi @dmuelle,
Can you help refine how the beta blog posts are promoted the beta table here, https://openliberty.io/start/#runtime_betas?

Here is my novice attempt 
![image](https://user-images.githubusercontent.com/31117513/170313030-a5089e3b-dea0-40e7-80fe-0b069adea2b6.png)

**Comment by dmuelle** [comment]
I think this looks good. A minor tweak for IBM Style:

Beta content might change from release to release. For more information, see our beta blog posts.

One question though- what do we mean by "Beta content might change from release to release" ? Doesn't the beta content almost always change for every release? Or are we trying to say that beta content might change during the course of a release? ie whats in the beta might change before GM? If the latter, we could be more specific. 

Happy to suggest revisions but just want to understand if Im reading that right

**Comment by kinueng** [comment]
RE: https://github.com/OpenLiberty/openliberty.io/issues/2579#issuecomment-1137786519

I do recall that it is rare but can happen where a beta doesn't have new content and so we use "might change".

Hi @mbroz2,
Can you help answer @dmuelle question about how the beta release changes? I do not know the answer

**Comment by Emily-Jiang** [comment]
> Hi @Emily-Jiang,
> 
> The Beta downloads page has a link to all the blogs related to the beta builds. The sentence `Beta content may change from release to release` is a link to the blog posts with the `beta` tag. This is the link https://openliberty.io/blog/?search=beta&key=tag.
> 
> The top blog post filtered by beta tag is for `22.0.0.5-beta`.
> 
> Would it help if we rework the `Beta content may change from release to release` to be more obvious the link takes you to beta blog posts?
> 
I don't think we need the text `Beta content may change from release to release` at all, because we only display one beta image. The link should just point to the latest beta blog post.

**Comment by mbroz2** [comment]
Liberty's zero migration dictates that once something is formally released, it doesn't change.  This is not the case for beta content, and that's what that sentence was trying to capture.  We could change that section to state the following, though it might be a bit too wordy and 'obvious' as we're pretty much defining what is commonly known to be beta content:

> Beta features and functions are made available to socialize what the Open Liberty team is actively developing and obtain early [feedback]{link}.  Beta content is subject to change and inclusion in a beta does not guarantee inclusion in a GA release.  For details regarding what each beta release includes, checkout the [beta blog posts]{link}.

Alternatively, the second sentence could read "Beta content may change before inclusion in a GA release, or never be released at all."

Or perhaps better still, reorder those sentences:
> Beta features and functions are made available to socialize what the Open Liberty team is actively developing and obtain early [feedback](link).  For details regarding what each beta release includes, checkout the [beta blog posts](link).  
> 
> Note: Beta content is subject to change and inclusion in a beta does not guarantee inclusion in a GA release.

**Comment by dmuelle** [comment]
Is "socialize" more of an internal IBM term? If so, maybe something like

> We publish beta releases to share and obtain early feedback on what the Open Liberty team is actively developing. Beta content is subject to change and inclusion in a beta does not guarantee inclusion in a GA release. For details about what each beta release includes, checkout the [beta blog posts](https://github.com/OpenLiberty/openliberty.io/issues/link).


Also- since there is only ever 1 beta available at a time, we might put the codesigning explanation after the table to avoid a big block of text that might make the beta seem more complex than it is. Just a thought.

**Comment by Emily-Jiang** [comment]
> Is "socialize" more of an internal IBM term? If so, maybe something like
> 
> > We publish beta releases to share and obtain early feedback on what the Open Liberty team is actively developing. Beta content is subject to change and inclusion in a beta does not guarantee inclusion in a GA release. For details about what each beta release includes, checkout the [beta blog posts](https://github.com/OpenLiberty/openliberty.io/issues/link).
> 
> Also- since there is only ever 1 beta available at a time, we might put the codesigning explanation after the table to avoid a big block of text that might make the beta seem more complex than it is. Just a thought.

Sounds good! I like the idea of using a codesigning explanation.
